Architecture.md

```diff
@@ -65,7 +69,36 @@
 
 #### Client sign-up and login process
 
-***TODO: [CLIENT] mermaid diagram and description of the login process flow, for both the CLI and the web frontend***
+##### Login process
+
+###### CLI
+
+In order to log in from the CLI tool, the Client must have first created an account on the web frontend since the CLI tool does not allow for account creation.
+This is intentional, and it ensures that creating an account cannot be automated. Using a website can also allow for more verifications, like [CAPTCHAs](https://en.wikipedia.org/wiki/CAPTCHA).
+
+The login process begins with the user typing the `paastech login` command which would prompt them to enter their email address and password. The CLI tool then sends a request to the API to get a JWT token.
+This token is stored locally in the user's config directory and is used for all subsequent commands requiring to be authenticated.
```

Comment:
Is the line return intentional ? Note that this doesn't go to the next line. Markdown interprets lines 79, 80 and 81 as the same paragraph.

Reply:
Line returns are intentional and help cut the paragraph into small pieces. This will be enforced by Prettier later on.